src/client/renderer/r_light.c

```diff
@@ -182,29 +185,19 @@ void R_EnableLights(uint64_t mask) {
 	if (mask) { // enable up to MAX_ACTIVE_LIGHT sources
 		const r_light_t *l = r_view.lights;
 
-		vec4_t position = { 0.0, 0.0, 0.0, 1.0 };
-		vec4_t diffuse = { 0.0, 0.0, 0.0, 1.0 };
-
 		for (uint16_t i = 0; i < r_view.num_lights; i++, l++) {
 
-			if (j == MAX_ACTIVE_LIGHTS)
+			if (j == r_state.max_lights)
```

Comment:
Sure, I'm just saying that the name is kind of misleading. The previous constant was named `MAX_ACTIVE_LIGHTS`, which is more accurate and doesn't conflate "how many lights can be in a scene" with "how many lights can I enable for a given primitive."